src/openrct2/localisation/StringIds.h

```diff
@@ -3938,9 +3938,14 @@ enum
     STR_MULTITHREADING_TIP = 6306,
 
     STR_TILE_INSPECTOR_COLOUR_SCHEME = 6307,
+    
+    STR_SCROLL_MODE = 6308,
+    STR_SCROLL_MODE_STANDARD = 6309,
+    STR_SCROLL_MODE_DISABLED = 6310,
+    STR_SCROLL_MODE_TIP = 6311,
 
     // Have to include resource strings (from scenarios and objects) for the time being now that language is partially working
-    STR_COUNT = 32768
+    STR_COUNT = 32771
```

Comment:
Please revert this change.

```suggestion
    STR_COUNT = 32769
```